xapian-applications/omega/templates/query

```diff
@@ -81,23 +85,30 @@ $if{$not{$msizeexact},
   All $nice{$msize} matches,
   $nice{$add{$topdoc,1}}$if{$ne{$add{$topdoc,1},$last},-$nice{$last}} of exactly $nice{$msize} matches}
 }
+</section>
 <hr>
 </center>
+
+<section>
 $list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;$nice{$freq{$_}}},Term frequencies: ,$. ,}
 <br><small>Search took $time seconds</small>
+</section>
 <table>
 $hitlist{<tr><td valign=top>
 ${<img src="/icons/omega/score-$div{$percentage,10}.png" alt="$percentage%" height=16 width=32>}
+
 <div title="$percentage%" style='float:left;width:60px;height:10px;border:1px solid black;margin-top:4px;'>
 <div style='width:$div{$mul{$percentage,6},10}px; height:10px; background-color: red;'>
 </div></div>
 <div style='float:left;margin-top:2px;font-size:x-small;'>
+<time>
```

Comment:
This is great, but I think `time` should replace the `span` below, and change `title` to [`datetime`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) (or possibly include both, since I suspect `datetime` doesn't provide a hover). Now that the datetime format for `datetime` is different to how we're using `title` at the moment.

Note that with Javascript enabled something else happens (to show a localised time), and you'll want to update `write_date()` accordingly.